src/clarity-angular/wizard/wizard-page.ts

```diff
@@ -119,6 +119,25 @@ export class WizardPage implements OnInit {
     @Output("clrWizardPagePreventDefaultCancelChange") stopCancelChange: EventEmitter <boolean> =
         new EventEmitter();
 
+    // overrides next from the page level, so you can use an alternate function for
+    // validation or data-munging with clrWizardPageOnNext
+    private _stopNext = false;
+    public get stopNext(): boolean {
+        return this._stopNext;
+    }
+    @Input("clrWizardPagePreventDefaultNext")
+    public set stopNext(val: boolean) {
+        let valBool = !!val;
+        if (valBool !== this._stopNext) {
+            this._stopNext = valBool;
+            this.stopNextChange.emit(valBool);
+        }
+        // SPECME
+    }
+
+    @Output("clrWizardPagePreventDefaultNextChange") stopNextChange: EventEmitter <boolean> =
+        new EventEmitter();
```

Comment:
This output doesn't make sense to me. What's a use case that would need it? From what I can see, the only way this is emitted is if the user gave us the corresponding input, which is a case where we _don't_ want to emit anything to avoid loops.